distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestFederation.java

```diff
@@ -1672,7 +1707,7 @@ private Action getCswRetrievalHeaders(String filename) {
     }
 
     private String getResourceData(String metacardId) {
-        return String.format("Data for metacard ID %s", metacardId);
+        return String.format("Hello there. This is data for metacard ID %s", metacardId);
```

Comment:
I would at least put a comment in there that you are trying to get to a specific amount of characters and the reason why. Otherwise, we could come up with a better test message. For example we can include "Mocked resource" or something along those lines in there.